examples/twittermap/web/public/javascripts/common/services.js

```diff
@@ -471,6 +529,17 @@ angular.module('cloudberry.common', ['cloudberry.mapresultcache'])
             break;
           case "batchPointMapRequest":
             break;
+          case "heat":
+            if(angular.isArray(result.value)) {
+              cloudberryService.commonTweetResult = result.value[0].slice(0, defaultSamplingSize - 1);
+              cloudberryService.heatmapMapResult = result.value[0];
+            }
+            break;
+          case "heatTime":
```

Comment:
Why this heatTime is necessary...

Reply:
It is there for the same reason as we have that in pinmap. We still need to update the time histogram when using heatmap. Should they share the same query? I think when we optimize pinmap and heatmap queries, their time histogram queries may also differ from each other, so I still keep them separate.

Reply:
OK